src/v/security/audit/audit_log_manager.h

```diff
@@ -106,6 +107,8 @@ class audit_log_manager
     /// Note does not include records already sent to client
     size_t pending_events() const { return _queue.size(); };
 
+    audit_probe& probe() { return *_probe; }
```

Comment:
nit: If this isn't publicly used i'd move it to the private section or just omit

Reply:
Nice, missed the `friend class audit_sink` 👍